code/modules/surgery/organs/internal/tongue/_tongue.dm

```diff
@@ -146,21 +143,27 @@
 	ADD_TRAIT(tongue_owner, TRAIT_AGEUSIA, NO_TONGUE_TRAIT)
 	tongue_owner.voice_filter = initial(tongue_owner.voice_filter)
 
-/obj/item/organ/internal/tongue/apply_organ_damage(damage_amount, maximum, required_organtype)
+/obj/item/organ/internal/tongue/apply_organ_damage(damage_amount, maximum = maxHealth, required_organ_flag)
 	. = ..()
 	if(!owner)
 		return
-	//tongues can't taste food when they are failing
+	apply_tongue_effects()
+
+/// Applies effects to our owner based on how damaged our tongue is
+/obj/item/organ/internal/tongue/proc/apply_tongue_effects()
```

Comment:
Already added in #76359 ? Weird it's still showing up

Reply:
I have no clue, but the code is correct regardless